packages/template-retail-react-app/package.json

```diff
@@ -14,7 +14,7 @@
     "build-translations": "npm run extract-default-translations && npm run compile-translations && npm run compile-translations:pseudo",
     "compile-translations": "node ./scripts/translations/compile-folder.js translations",
     "compile-translations:pseudo": "node ./scripts/translations/compile-pseudo.js translations/en-US.json",
-    "extract-default-translations": "node ./scripts/translations/extract-default-messages.js en-US",
+    "extract-default-translations": "node ./scripts/translations/extract-default-messages.js en-US && node ./scripts/translations/extract-default-messages.js en-GB",
```

Comment:
Since we use both en-US and en-GB for English, adding one more extraction command for en-GB can avoid the issue of missing translation for en-GB